cmake/Modules/CMakePolicy.cmake

```diff
@@ -12,7 +12,13 @@ if(POLICY CMP0116)
 endif()
 
 +# Enable debug info compatible with SCCache
-+# https://github.com/mozilla/sccache?tab=readme-ov-file#usage
+# MSVC debug information format flags are selected by an
+# abstraction. New in CMake 3.25.
```

Comment:
While it's not the official wording, what about "MSVC debug information format flags are selected via `CMAKE_MSVC_DEBUG_INFORMATION_FORMAT`, instead of embedding flags in e.g. `CMAKE_CXX_FLAGS_RELEASE`"? That's much clearer IMO.